docs/assemblies/proc_creating-a-set-of-data-plane-nodes.adoc

```diff
@@ -291,6 +292,20 @@ spec:
   		  edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified"
   		  edpm_neutron_sriov_image: "quay.io/podified-antelope-centos9/openstack-neutron-sriov-agent:current-podified"
   		  edpm_multipathd_image: "quay.io/podified-antelope-centos9/openstack-multipathd:current-podified"
+endif::[]
+ifeval::["{build}" == "downstream"]
+        edpm_iscsid_image: "registry.redhat.io/rhosp-dev-preview/openstack-iscsid:18.0"
+  		  edpm_logrotate_crond_image: "registry.redhat.io/rhosp-dev-preview/openstack-cron:18.0"
+  		  edpm_ovn_controller_agent_image: "registry.redhat.io/rhosp-dev-preview/openstack-frr:18.0"
+  		  edpm_ovn_metadata_agent_image: "registry.redhat.io/rhosp-dev-preview/openstack-neutron-metadata-agent-ovn:18.0"
+  		  edpm_frr_image: "registry.redhat.io/rhosp-dev-preview/openstack-frr:18.0"
+  		  edpm_ovn_bgp_agent_image: "registry.redhat.io/rhosp-dev-preview/openstack-ovn-bgp-agent:18.0"
+  		  telemetry_node_exporter_image: "quay.io/prometheus/node-exporter:v1.5.0"
+  		  edpm_libvirt_image: "registry.redhat.io/rhosp-dev-preview/openstack-nova-libvirt:18.0"
+  		  edpm_nova_compute_image: "registry.redhat.io/rhosp-dev-preview/openstack-nova-compute:18.0"
+  		  edpm_neutron_sriov_image: "registry.redhat.io/rhosp-dev-preview/openstack-neutron-sriov-agent:18.0"
+  		  edpm_multipathd_image: "registry.redhat.io/rhosp-dev-preview/openstack-multipathd:18.0"
+endif::[]
```

Comment:
Hey, does the indenting need fixed from line 298? If so, also needs fixed for the upstream block.